src/jdk.jshell/share/classes/jdk/internal/jshell/tool/JShellToolBuilder.java

```diff
@@ -215,6 +217,16 @@ public JavaShellToolBuilder interactiveTerminal(boolean terminal) {
         return this;
     }
 
+    @Override
+    public JavaShellToolBuilder windowSize(int columns, int rows) {
+        if (columns <= 0)
+            throw new IllegalArgumentException("columns = " + columns);
+        if (rows <= 0)
+            throw new IllegalArgumentException("-columns = " + -columns);
```

Comment:
Is this supposed to have a message like `"rows = " + rows` instead

Reply:
Oops, yes - thanks. Fixed in 08f7d3bc069.